README.md

```diff
@@ -283,6 +261,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Hi, I would advise you to link MIT to the license. Tho it is part of the project requirements, it makes your project more professional and be trusted

Reply:
@ olipliche 
I am not impress with your review, you took over 1 hour and yet disapprove a project which has no errors, the required changes as you mentioned is not part of the project requirement.

Reply:
Hi, Thank you for reaching out. I mentioned in my comment that it is part of the general requirements. The general requirements belong to your project. We request this not block the students. Your journey is not only about coding but also about teaching you to become more professional. Sorry for any disagreements. All the best